### PR TITLE
release: release 2.0-rc version.

### PR DESCRIPTION
**Pull request by moonming**
https://github.com/apache/apisix/milestone/7
2.0 milestone almost finished, it’s time to start release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,31 @@
...
+
+## Plugin
+- :sunrise: **add AK/SK(HMAC) auth plugin.**[#2192](https://github.com/apache/apisix/pull/2192)
+- :sunrise: add referer-restriction plugin.[#2352](https://github.com/apache/apisix/pull/2352)
+- `limit-count` support to use `redis` cluster.[#2406](https://github.com/apache/apisix/pull/2406)
+- feat(proxy-cache): store the temporary file under cache directory. [#2317](https://github.com/apache/apisix/pull/2317)
+- feat(http-logger): support for specified the log formats via admin API [#2309](https://github.com/apache/apisix/pull/2309)
+
+## Bugfix
+- :bug: **`high priority`** When the data plane receives an instruction to delete a resource(router, upstream etc.), it does not properly clean up the cache, resulting in the existing resources cannot be found. This problem only occurs in the case of long and frequent deletion operations.[#2168](https://github.com/apache/apisix/pull/2168)
```
Comment (juzhiyuan) [inline CHANGELOG.md:53]: upstream  and so on or upstream, etc.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,31 @@
...
+- fix routing priority does not take effect.[#2447](https://github.com/apache/apisix/pull/2447)
+- set random seed for each worker process at `init_worker` phase, only `init` phase is not enough.[#2357](https://github.com/apache/apisix/pull/2357)
+- remove unsupported algorithm in jwt plugin.[#2356](https://github.com/apache/apisix/pull/2356)
+- return right response code when `http_to_https` enabled in redirect plugin.[#2311](https://github.com/apache/apisix/pull/2311)
```
Comment (juzhiyuan) [inline CHANGELOG.md:57]: correct response code

File: CHANGELOG_CN.md [hunk 3]
```diff
@@ -31,6 +32,29 @@
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+### Core
+- :sunrise: **从 etcd v2 协议迁移到 v3，这是不向下兼容的修改。Apache APISIX 只支持 etcd 3.4 以及后续的版本。** [#2036](https://github.com/apache/apisix/pull/2036)
```
Comment (juzhiyuan) [inline CHANGELOG_CN.md:36]: missing Apache in English version

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

**Comment by moonming** [comment]
@juzhiyuan fixed, please take a look

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by membphis — APPROVED** [review]

**Pull request merged**